app/workers/evss/disability_compensation_form/submit_form526.rb

```diff
@@ -100,7 +100,16 @@ def forward_to_mas?(submission)
         submission.diagnostic_codes.size == 1 &&
           RapidReadyForDecision::Constants::MAS_DISABILITIES.include?(submission.diagnostic_codes.first) &&
           submission.disabilities.first['disabilityActionType']&.upcase == 'INCREASE' &&
-          !submission.pending_eps?
+          !submission.pending_eps? &&
+          !disability_not_service_connected?
```

Comment:
Since this worker runs on all Form526 submissions, I wanted all the easy checks to short-circuit this MAS-eligibility method before making the EVSS call.